terraform/graph_config_node.go

```diff
@@ -45,634 +39,3 @@ type GraphNodeTargetable interface {
-	if raw := count.Raw[count.Key]; raw != "1" {
-		return true
-	}
-
-	// Okay, we're dealing with a static count. There are a few ways
-	// to include this resource.
-	prefix := n.Original.Resource.Id()
-
-	// If we're present in the diff proper, then keep it.
-	if d != nil {
-		for k, _ := range d.Resources {
-			if strings.HasPrefix(k, prefix) {
-				return true
-			}
-		}
-	}
-
-	// If we're in the state as a primary in any form, then keep it.
-	// This does a prefix check so it will also catch orphans on count
-	// decreases to "1".
-	if s != nil {
-		for k, v := range s.Resources {
-			// Ignore exact matches
-			if k == prefix {
-				continue
-			}
-
-			// Ignore anything that doesn't have a "." afterwards so that
-			// we only get our own resource and any counts on it.
-			if !strings.HasPrefix(k, prefix+".") {
-				continue
-			}
-
-			// Ignore exact matches and the 0'th index. We only care
-			// about if there is a decrease in count.
-			if k == prefix+".0" {
-				continue
-			}
-
-			if v.Primary != nil {
-				return true
-			}
-		}
-
-		// If we're in the state as _both_ "foo" and "foo.0", then
-		// keep it, since we treat the latter as an orphan.
-		_, okOne := s.Resources[prefix]
-		_, okTwo := s.Resources[prefix+".0"]
-		if okOne && okTwo {
-			return true
-		}
-	}
-
-	return false
-}
-
-// graphNodeModuleExpanded represents a module where the graph has
-// been expanded. It stores the graph of the module as well as a reference
-// to the map of variables.
-type graphNodeModuleExpanded struct {
-	Original    dag.Vertex
-	Graph       *Graph
-	InputConfig *config.RawConfig
-
-	// Variables is a map of the input variables. This reference should
-	// be shared with ModuleInputTransformer in order to create a connection
-	// where the variables are set properly.
-	Variables map[string]string
-}
-
-func (n *graphNodeModuleExpanded) Name() string {
-	return fmt.Sprintf("%s (expanded)", dag.VertexName(n.Original))
-}
-
-func (n *graphNodeModuleExpanded) ConfigType() GraphNodeConfigType {
-	return GraphNodeConfigTypeModule
-}
-
-// GraphNodeDotter impl.
-func (n *graphNodeModuleExpanded) DotNode(name string, opts *GraphDotOpts) *dot.Node {
-	return dot.NewNode(name, map[string]string{
-		"label": dag.VertexName(n.Original),
-		"shape": "component",
-	})
-}
-
-// GraphNodeEvalable impl.
-func (n *graphNodeModuleExpanded) EvalTree() EvalNode {
-	var resourceConfig *ResourceConfig
-	return &EvalSequence{
-		Nodes: []EvalNode{
-			&EvalInterpolate{
-				Config: n.InputConfig,
-				Output: &resourceConfig,
-			},
-
-			&EvalVariableBlock{
-				Config:    &resourceConfig,
-				Variables: n.Variables,
-			},
-
-			&EvalOpFilter{
-				Ops: []walkOperation{walkPlanDestroy},
-				Node: &EvalSequence{
-					Nodes: []EvalNode{
-						&EvalDiffDestroyModule{Path: n.Graph.Path},
-					},
-				},
-			},
-		},
-	}
-}
-
-// GraphNodeSubgraph impl.
-func (n *graphNodeModuleExpanded) Subgraph() *Graph {
-	return n.Graph
-}
```

Comment:
With the file split it's sorta hard to read for differences in the graph nodes - can you highlight the important areas where the node code changes for me? <3